generators/server/index.js

```diff
@@ -360,6 +360,7 @@ module.exports = class extends BaseGenerator {
 
             saveConfig() {
                 this.config.set('jhipsterVersion', packagejs.version);
+                this.config.set('applicationType', this.applicationType);
```

Comment:
if this is added here then you need to do it for client sub gen as well

Reply:
yes, you're right. I will do it later